turbinia/api/utils.py

```diff
@@ -189,3 +202,36 @@ async def create_tarball(output_path: str) -> AsyncGenerator[bytes, Any]:
 
     # Yield end-of-archive marker.
     yield stream.pop()
+
+
+def tail_log(log_path, max_lines=500) -> str:
```

Comment:
Very professional. Or you use subprocess and tail.....

Reply:
Got to love re-inventing the wheel :)